packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -400,11 +266,11 @@ function TaxFields( { section, taxInfo, setters, CountrySelectMenu, countriesLis
 					type="text"
 					label={ isZip ? translate( 'Zip code' ) : translate( 'Postal code' ) }
 					value={ postalCode.value }
-					onChange={ value =>
-						setContactField( 'postalCode', { value, isTouched: true, isValid: !! value } )
-					}
+					onChange={ event => {
```

Comment:
I think that `Field`'s `onChange` function already pulls out the `value` from the event object.